<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="toolButtonAddBody">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../data/icons.qrc">
               <normaloff>:/icons/car--plus.png</normaloff>:/icons/car--plus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonAddStaticBody">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../data/icons.qrc">
               <normaloff>:/icons/wall--plus.png</normaloff>:/icons/wall--plus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../data/icons.qrc">
               <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>76</width>
           <height>367</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="Canvas" name="canvasWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="PropertyBrowser" name="propertyBrowser" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Actual"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionGrid_draw"/>
   <addaction name="actionGrid_snap"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionTool_edit"/>
   <addaction name="actionTool_box"/>
   <addaction name="actionTool_circle"/>
   <addaction name="actionTool_polygon"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/magnifier-zoom-in.png</normaloff>:/icons/magnifier-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Increment zoom</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/magnifier-zoom-out.png</normaloff>:/icons/magnifier-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Decrement zoom</string>
   </property>
  </action>
  <action name="actionZoom_Actual">
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/magnifier-zoom-actual.png</normaloff>:/icons/magnifier-zoom-actual.png</iconset>
   </property>
   <property name="text">
    <string>Actual Zoom</string>
   </property>
   <property name="toolTip">
    <string>Set zoom 1:1</string>
   </property>
  </action>
  <action name="actionGrid_draw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/grid.png</normaloff>:/icons/grid.png</iconset>
   </property>
   <property name="text">
    <string>Draw grid</string>
   </property>
   <property name="toolTip">
    <string>Draw grid</string>
   </property>
  </action>
  <action name="actionGrid_snap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/grid-snap.png</normaloff>:/icons/grid-snap.png</iconset>
   </property>
   <property name="text">
    <string>Snap to grid</string>
   </property>
   <property name="toolTip">
    <string>Snap to grid</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionTool_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/arrow.png</normaloff>:/icons/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Edit tool</string>
   </property>
  </action>
  <action name="actionTool_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/box.png</normaloff>:/icons/box.png</iconset>
   </property>
   <property name="text">
    <string>Box tool</string>
   </property>
  </action>
  <action name="actionTool_circle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/circle.png</normaloff>:/icons/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle tool</string>
   </property>
   <property name="toolTip">
    <string>Circle tool</string>
   </property>
  </action>
  <action name="actionTool_polygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../data/icons.qrc">
     <normaloff>:/icons/polygon.png</normaloff>:/icons/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon tool</string>
   </property>
   <property name="toolTip">
    <string>Polygon tool</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PropertyBrowser</class>
   <extends>QWidget</extends>
   <header>propertybrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
